recipe/meta.yaml

```diff
@@ -14,11 +14,11 @@ source:
 
 build:
   number: 0
-  skip: True # [py<36]
-  script: "{{ PYTHON }} -m pip install --no-deps -vv ."
+  skip: True # [py<38]
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation -vv .
   entry_points:
     - black = black:patched_main
-    - blackd = blackd:patched_main
+    - blackd = blackd:patched_main [d]
```

Comment:
What's the `[d]` for? I have never seen that syntax before.

Reply:
I will remove it. It is to indicate that it requires the [d] optional dependencies (aiohttp). 
The code doesn't process anything of this.